lib/active_model/serializer.rb

```diff
@@ -476,9 +479,7 @@ def scope
 
     def _serializable_hash
       return nil if @object.nil?
-      @node = attributes
```

Comment:
Ok, thanks. I'm not as familiar with the 0.8 code flow.

Do you think you can get CI passing?